action.yml

```diff
@@ -16,6 +16,10 @@ inputs:
     description: 'add https://oss.sonatype.org/content/repositories/snapshots to repository list - true or false'
     default: "false"
     required: false
+  artifactory_org:
+    description: 'enable jfrog default profiles and set jfrog artifactory org name'
+    default: "myorg"
```

Comment:
What happens when user not provide settings for this, I think this value will be used and artifactory feature will be configured in settings.xml
Better will be without default.